src/style.css

```diff
@@ -0,0 +1,8 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+* {
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Good job 👌but as it's mentioned in the project requirements you're supposed to design your to-do list similar to the one in this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg&ab_channel=EmiliaAndrzejewska). So please make sure to design your application using CSS.

![image](https://user-images.githubusercontent.com/34779928/200926560-6c8f60d9-5550-4ff4-83b4-1b93e63683c6.png)

Reply:
Hey @aimalamiri , If i rename the branch it will close the pull request, is that okay?

Reply:
No, you don't need to rename the branch for this pull request, that's why I put that request under the optional section. But I forget to add the `[Optional]` Prefix. Sorry about that.

Reply:
It's all right, is the styling css also an optional change?

Reply:
- Please style your project to look like the wireframe, just as the previous reviewer suggested. Only comments with the `OPTIONAL` tag prefix can be considered optional changes. Please fix this issue 😄 

Your project:
<img width="959" alt="wwdwd" src="https://user-images.githubusercontent.com/73607512/201097310-b0bb6be6-76dc-40fb-aa41-83d9f5f7d1bb.PNG">